services/credentials.go

```diff
@@ -28,30 +28,30 @@ const (
 	credsRequestMaxAge = time.Duration(15) * time.Minute
 )
 
-type quota struct {
+type Quota struct {
```

Comment:
exporting this type has a somewhat unwieldy consequence- the `map[credentials.OperatorType]Quota` var is static, but mutable. A user of the package could subsequently accidentally modify global quota from their const values.

Unfortunately, I think it's worth being defensive here... Can write a package-scoped function that returns a [json.RawMessage](https://pkg.go.dev/encoding/json#RawMessage) instead? Then we can define something like `services.GetQuotaJSON(ot OperatorType) *json.RawMessage` and use the result on the response.

Reply:
I knew there had to be a better way! This makes much more sense. Easier change than I anticipated too.